Docker/Dockerfile

```diff
@@ -23,18 +23,6 @@ ARG FRESHRSS_VERSION
 ARG SOURCE_BRANCH
 ARG SOURCE_COMMIT
 
-LABEL \
-	org.opencontainers.image.authors="Alkarex" \
-	org.opencontainers.image.description="A self-hosted RSS feed aggregator" \
-	org.opencontainers.image.documentation="https://freshrss.github.io/FreshRSS/" \
-	org.opencontainers.image.licenses="AGPL-3.0" \
-	org.opencontainers.image.revision="${SOURCE_BRANCH}.${SOURCE_COMMIT}" \
-	org.opencontainers.image.source="https://github.com/FreshRSS/FreshRSS" \
-	org.opencontainers.image.title="FreshRSS" \
-	org.opencontainers.image.url="https://freshrss.org/" \
-	org.opencontainers.image.vendor="FreshRSS" \
-	org.opencontainers.image.version="$FRESHRSS_VERSION"
```

Comment:
I would prefer to keep the static ones here, so that they exist when the builds are done locally, e.g. like
https://github.com/FreshRSS/FreshRSS/blob/8b2f7848eb1054ee26048e3ffe76855410286407/Docker/freshrss/docker-compose.yml#L12-L15

`authors` can be removed

By the way, if anyone knows a way to populate `revision` and/or `version` even in those more manual scenarios, let me know

Reply:
not sure what happen when you use the github action ones and the static ones.

Reply:
Let's try to remove the static ones from GitHub Actions, so that we can see whether that is sufficient or whether they need to be overridden again

Reply:
I found some docu about. 
passing labels at build time will overwrite the static ones